data/json/items/armor/ballistic_armor.json

```diff
@@ -111,13 +111,13 @@
     "id": "stab_plate",
     "type": "ARMOR",
     "category": "armor",
-    "name": { "str": "soft stab plate" },
+    "name": { "str": "soft stab panel" },
     "//": "based on https://www.highcomarmor.com/product/trooper-hcspike3/",
-    "description": "A soft armor insert designed to protect against stabbing.  Not great protection but incurs basically no additional encumbrance.  This is intended to be worn in a ballistic vest.",
+    "description": "A soft armor insert designed to protect against stab wounds.  While it's not great protection, it incurs no additional encumbrance.  This is intended to be inserted into a ballistic vest, and can't be worn on its own.",
```

Comment:
There are two flags that basically give all the information from the last sentence. I realize this is my mistake as well adding the "This is intended to be inserted into a ballistic vest"

![image](https://user-images.githubusercontent.com/4514073/162961217-9396f513-ada2-41d3-b40b-f8fb149ecbc1.png)

Reply:
I'm cool keeping the information shown in both the description and flags, as I think it presents the info in a more digestible way for players just now finding the item. I'm also cool with axing it, though, since it's a feature you implemented and you might have a better sense of it than I do. What do you feel would work best?

Reply:
As someone who answers a lot of "how do ballistic plates work questions" I think more explanation is better but I've gotten push back from the translators in the past as well. On thinking about it I'd say keep it in. Sorry for being wishy washy.

Reply:
hey, it's fine. all writing style changes are subjective, so I really don't mind hearing either side of an argument. worst comes to worst, we can always patch it out down the line, no harm done.

Reply:
That was my thought too, you've already got it done and we both think it's a good idea, no reason for me to play some malformed devils advocate 😆 

So for sure ignore this one.